packages/codemirror/src/codemirror.js

```diff
@@ -300,43 +303,69 @@ export function createCypherEditor(parentDOMElement, options = {}) {
     fire: fireKeyDown
   } = createEventHandlers();
 
+  const { on: onKeyUp, off: offKeyUp, fire: fireKeyUp } = createEventHandlers();
+
   const lineNumberClick = (line, event) => {
     fireLineNumberClick(line, event);
   };
 
   const keyDown = (event) => {
+    pressedKey =
+      !event.metaKey &&
+      !event.ctrlKey &&
+      !event.altKey &&
+      event.key &&
+      event.key.length === 1
```

Comment:
I wonder if this will miss stuff like emojis or other unicode characters 🤔 Probably not an issue though

Reply:
good point, probably safer to use `event.key.length > 0 && event.key.length < 3` in case unicode characters are pressed that have length 2

Reply:
actually, I tried out unicode characters, and they work completely differently at least on mac. So yeah, we only support non-unicode characters for autocomplete trigger strings.